ui/jobs/components/util.js

```diff
@@ -0,0 +1,93 @@
+// import ResourceBar from '../../../resources/resourcebar';
```

Comment:
Is there a reason to not just make this file typescript to start with?

Reply:
I wanted to write TypeScript, but there are typing issues cannot be fixes right now, so commented out temporary.